### PR TITLE
release 1.0.1 note

### PR DESCRIPTION
**Pull request by linsun**
1st draft, pls comment, thanks @geeknoid @costinm @rshriram @sdake and others.

### CODE REVIEW

File: content/about/notes/1.0.1/index101.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+weight: 91
+icon: /img/notes.svg
+---
+
+We're proud to release Istio 1.0.1! Within this release, we have been addressing some critical issues found by the community when using Istio 1.0.
+
+These release notes describe what's different between Istio 1.0 and Istio 1.0.1. 
+
+## Networking
+- Pilot scability and Envoy sidecar startup improvement.
```
Comment (gyliu513) [inline content/about/notes/1.0.1/index101.md]: Is it possible that we add the issue or PR link here for each item here? just like a CHANGELOG?

Comment (linsun) [reply content/about/notes/1.0.1/index101.md]: We haven't done that for any of our prior release notes other than for some of the known issues.  I'm not sure if it is worth our time to get it matched correctly for each items, @geeknoid thoughts?

File: content/about/notes/1.0.1/index101.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+- Fixed virtual service host mismatch issue when port is added.
+- Allow multiple destination rules with different subsets for the same host.
+- Ability to merge non conflict http routes in multiple virtual service resources that are bounded to the same gateway resource.
```
Comment (frankbu) [inline content/about/notes/1.0.1/index101.md]: Replace the above 2 bullets with the following:

* Added limited support for [merging multiple virtual service or destination rule definitions](/help/ops/traffic-management/deploy-guidelines/#multiple-virtual-services-and-destination-rules-for-the-same-host) for the same host.

File: content/about/notes/1.0.1/index101.md (hunk 1 above)
Comment (geeknoid) [reply content/about/notes/1.0.1/index101.md]: I don't want to do this. These are customer-centric release notes, not developer-centric.

I will be shortly adding a standard feature to all release note pages which provides a link to the binary, along with a link to a GitHub diff page for the patch.

Comment (gyliu513) [reply content/about/notes/1.0.1/index101.md]: Another point I want to mention is for future release, we should add `release-note` for some important PRs and then use `release-note` to generate CHANGELOG by following k8s way, this can make people trace issues and change easily.

File: content/about/notes/1.0.1/index.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+---
+title: Istio 1.0.1
+weight: 91
+icon: /img/notes.svg
+---
+
+We're proud to release Istio 1.0.1! Within this release, we have been addressing some critical issues found by the community when using Istio 1.0.
```
Comment (frankbu) [inline content/about/notes/1.0.1/index.md]: Unlike 1.0, this release is not such a big deal, so I would remove the first sentence: We’re proud to release Istio 1.0.1!

Comment (linsun) [reply content/about/notes/1.0.1/index.md]: are we not proud of it? :). i can remove it.

**Comment by linsun** [comment]
all issues are addressed @geeknoid PTAL

File: content/about/notes/1.0.1/index.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
+---
+title: Istio 1.0.1
+weight: 91
+icon: /img/notes.svg
+---
+
+Within this release, we have been addressing some critical issues found by the community when using Istio 1.0.
```
Comment (geeknoid) [inline content/about/notes/1.0.1/index.md]: Within this release, we have been addressing some critical -> This release address critical issues

File: content/about/notes/1.0.1/index.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+This release note describes what's different between Istio 1.0 and Istio 1.0.1.
+
+## Networking
+
+- Pilot scalability and Envoy sidecar startup improvement.
```
Comment (geeknoid) [inline content/about/notes/1.0.1/index.md]: - Improved Pilot scalabilty and Envoy startup time

File: content/about/notes/1.0.1/index.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+- Fixed virtual service host mismatch issue when port is added.
+- Added limited support for [merging multiple virtual service or destination rule definitions](/help/ops/traffic-management/deploy-guidelines/#multiple-virtual-services-and-destination-rules-for-the-same-host) for the same host.
+- Allow [outlier](https://www.envoyproxy.io/docs/envoy/latest/api-v1/cluster_manager/cluster_outlier_detection.html) consecutive gateway failures when using HTTP.
+
+## Environment
+
+- Ability to run Pilot only for users who wants to leverage traffic management feature of Istio.
```
Comment (geeknoid) [inline content/about/notes/1.0.1/index.md]: - Make is possible to use Pilot separate from the rest of Istio, for those users that want to only leverage Istio's traffic management functionality,

File: content/about/notes/1.0.1/index.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+- Convenient values-istio-gateway.yaml for users to run standalone gateways.
```
Comment (geeknoid) [inline content/about/notes/1.0.1/index.md]: Introduce the convenient `values-istio-gateway.yaml` configuration that enables users to run standalone gateways.

File: content/about/notes/1.0.1/index.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+- Various fixes for our helm based installation, including a prior issue with istio-sidecar-injector configmap not found.
```
Comment (geeknoid) [inline content/about/notes/1.0.1/index.md]: Fix a variety of Helm installation issues, including an issue with the `istio-sidecar-injector` config map not being found.

File: content/about/notes/1.0.1/index.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+- Fixed the Istio installation error with endpoint galley not ready.
```
Comment (geeknoid) [inline content/about/notes/1.0.1/index.md]: ...error with Galley not being ready.

File: content/about/notes/1.0.1/index.md [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+- Various mesh expansion fixes.
```
Comment (geeknoid) [inline content/about/notes/1.0.1/index.md]: Fixed a variety of issues around mesh expansion.

File: content/about/notes/1.0.1/index.md [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+- Added an experimental metrics expiration configuration to the Mixer Prometheus adapter.
+- Updated Grafana to 5.2.2.
+
+### Adapters
+
+- Ability to specify sink options for the Stackdriver adapter.
+
+## Galley
+
+- Configuration  validation improvement for health checks.
```
Comment (geeknoid) [inline content/about/notes/1.0.1/index.md]: Improved configuration validation for health checking.

File: content/help/ops/traffic-management/deploy-guidelines/index.md [hunk 12]
```diff
@@ -6,6 +6,118 @@ weight: 5
 
 This section provides specific deployment or configuration guidelines to avoid networking or traffic management issues.
 
+## Multiple virtual services and destination rules for the same host
```
Comment (geeknoid) [inline content/help/ops/traffic-management/deploy-guidelines/index.md]: Could you put this in a separate PR?

File: content/about/notes/1.0.1/index.md (hunk 11 above)
Comment (linsun) [reply content/about/notes/1.0.1/index.md]: Addressing all your comments... I kept health checks as it reads better to me.  for example:  https://cloudplatform.googleblog.com/2018/05/Kubernetes-best-practices-Setting-up-health-checks-with-readiness-and-liveness-probes.html

**Comment by linsun** [comment]
https://github.com/istio/istio.github.io/pull/2443/files should fix the link error

**Comment by geeknoid** [comment]
I just merged my changes to add the release note links, so you'll need to do a simple rebase to update.

Once all this lands in master, I'll merge the updated notes to the 1.0 branch and send announcements about 1.0.1 to all necessary channels.

**Comment by linsun** [comment]
@geeknoid should be good now, PTAL.  link issue is fixed in #2443
